eCourse_backend/courses/urls.py

```diff
@@ -26,7 +26,7 @@
     path('iframes/studentlist/<id>', course_student_list_iframe,
          name='course_studentlist_iframe'),
     # admin: create course
-    path('create/', create_course, name="create_course"),
+    path('create/', create_course_admin, name="create_course"),
```

Comment:
please revert renaming (i think it is the same as in the pr lecturer-and-student-site-2)